nixpkgs_review/builddir.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, name: str) -> None:
         os.environ["NIX_PATH"] = self.nixpkgs_path()
 
     def nixpkgs_path(self) -> str:
-        return f"nixpkgs={self.worktree_dir}:nixpkgs-overlays={self.overlay.path}"
+        return f"nixpkgs={self.worktree_dir} nixpkgs-overlays={self.overlay.path}"
```

Comment:
Is space an allowed separator in the environment variable as well?

Reply:
reading the source code, I don't think it is

Reply:
I rewrote the patch to make nixpkgs-review not rely on NIX_PATH, hopefully I didn't accidently break something

Reply:
I actually find it quite handy to have a NIX_PATH set. Can we not just keep it, but not rely on it for our own tooling?